docs/docs/concepts/authorization-tokens.mdx

```diff
@@ -12,46 +12,21 @@ There are a number of ways in which you can authenticate and authorize with Micr
+CLI for Microsoft 365 gets access to Microsoft 365 through your own Microsoft Entra application. 
 
 :::info
 
-When you decide to use your own Microsoft Entra application, you need to choose the application to be a **public client**. Despite the setting's description, the application will not be publicly accessible. This setting enables the use of the device flow for your own application. Without activating this setting, it is not possible to complete the authentication process. The option is currently only available in the preview blade for managing for Microsoft Entra applications.
+When you create your own Microsoft Entra application, you need to choose the application to be a **public client**. Despite the setting's description, the application will not be publicly accessible. This setting enables the use of the device flow for your own application. Without activating this setting, it is not possible to complete the authentication process. The option is currently only available in the preview blade for managing for Microsoft Entra applications.
 
 <br/>
 
 [![The 'public client' enabled for a Microsoft Entra application](../images/activate-public-client-aad-app.png)](../images/activate-public-client-aad-app.png)
 
 :::
 
-When specifying a custom Microsoft Entra application to be used by the CLI for Microsoft 365, set the `CLIMICROSOFT365_ENTRAAPPID` environment variable to the ID of your Microsoft Entra application.
-
-CLI for Microsoft 365 requires the following permissions to Microsoft 365 services:
-
-- Microsoft 365 SharePoint Online
-  - Have full control of all site collections
-  - Read user profiles
-  - Read and write managed metadata
-- Microsoft Graph
-  - Read and write to all app catalogs
-  - Read and write all groups
-  - Read and write directory data
-  - Access directory as the signed in user
-  - Read and write identity providers
-  - Read and write access to user mail
-  - Send mail as a user
-  - Read all usage reports
-  - Manage user's installed Teams apps
-  - Invite guest users to the organization
-  - Create, read, update and delete user tasks and projects
-- Microsoft 365 Management APIs
-  - Read service health information for your organization
-- Windows Azure Active Directory
-  - Access the directory as the signed-in user
-- Windows Azure Service Management API
-  - Access Azure Service Management as organization users
-- Viva Engage
-  - Read and write to the Viva Engage platform
+When specifying a Microsoft Entra application to be used by the CLI for Microsoft 365, set the `CLIMICROSOFT365_ENTRAAPPID` environment variable to the ID of your Microsoft Entra application.
+
+For a streamlined setup, you can use the [m365 setup](../cmd/setup.mdx) command to create a new app registration with all the necessary permissions to fully use the CLI for Microsoft 365. This command simplifies the process by automatically configuring the app registration with the required permissions.
```

Comment:
We're using "application" and "app registration" interchangeably. I suggest we consider picking one and using consistently throughout to avoid confusion.